### ISSUE TITLE
Error while running piqa_evaluate.py

### ISSUE THREAD

**Issue by jhyuklee** [uwnlp/piqa#4]
> $ python piqa_evaluate.py $SQUAD_DEV_PATH /tmp/piqa/context_emb/ /tmp/piqa/question_emb/
> Traceback (most recent call last):
>  File "piqa_evaluate.py", line 151, in <module>
>    progress=args.progress)
>  File "piqa_evaluate.py", line 123, in get_predictions
>   m = sim.max(1)
>  File "/home/jinhyuk/anaconda3/envs/p3.6/lib/python3.6/site-packages/numpy/core/_methods.py", line 28, in _amax
>   return umr_maximum(a, axis, None, out, keepdims, initial)
>numpy.core._internal.AxisError: axis 1 is out of bounds for array of dimension 1

Changing codes from

https://github.com/uwnlp/piqa/blob/c6e871919a6c664d53db58fd2d7c047843ce2e75/piqa_evaluate.py#L119-L125
to 
```
119         else:                                                                                                
120             q_emb = q_emb['arr_0']                                                                           
121             c_emb = c_emb['arr_0']                                                                           
122             m = np.matmul(c_emb, q_emb.T)                                                                  
123             # m = sim.max(1)                                                                                 
124                                                                                                              
125         argmax = m.argmax(0)
```
simply solves the problem, but don't know why the original code was in that form.
q_emb.shape = (1024,), c_emb.shape = (1008, 1024) leads to sim.shape = (1008),
and just taking argmax(0) seems fine.

**Comment by seominjoon** [comment]
q_emb.shape is intended to be (1, 1024) (for various reasons); is yours (1024,)?

**Comment by jhyuklee** [comment]
Yes, q_emb.shape was (1024,).

**Comment by seominjoon** [comment]
Bug fixed at 8228ccdc92803d30d1d8bdd2b156ef4497de175a
Please let me know if there is still an issue.

FYI, the reason for (1, 1024) instead of (1024,):

First, `scipy.sparse.save_npz` only supports saving matrix (2D), and does not support vector (1D). So when saving sparse vector, it anyway has to be (1, 1024), and for consistency, I am also enforcing dense (numpy) to be 2-dim.

 Second, I am thinking about allowing multiple query vectors (and this is why I am taking `max` instead of squeezing). Probably not immediately (in which case I will change it to squeeze), but it might happen in future so it will be more flexible and consistent to start with (1, 1024) than (1024,).

**Comment by jhyuklee** [comment]
https://github.com/uwnlp/piqa/commit/8228ccdc92803d30d1d8bdd2b156ef4497de175a clearly solves the issue. Thanks.
